lib/drush-node.js

```diff
@@ -65,40 +68,78 @@ Drush.version = function () {
  *
  * @return Promise
  */
-Drush.exec = function (command, options) {
+Drush.exec = function (args, options) {
   options = options || {};
+  if (typeof args === 'string') {
+    args = args.split(' ');
```

Comment:
Hooray for those looking the same... the point is different whitespace would render different results. Also something like `-a "wrapped string"` would cause problems.